nemo_text_processing/text_normalization/es/graph_utils.py

```diff
@@ -158,15 +158,25 @@ def roman_to_int(fst: 'pynini.FstLike') -> 'pynini.FstLike':
         fst: Any fst. Composes fst onto Roman conversion outputs.
     """
 
-    def _load_roman(file: str):
+    def _load_roman(file: str, upper_casing: bool):
```

Comment:
if you're adding the flag you can remove all upper case romans in your data file.

Reply:
I think all are lower-case except for the ones listed in `roman_exceptions.tsv`.  There I only removed the two-char upper-case lower-case strings, as the change will for sure deem them unnecessary.  I'd rather not touch the other exceptions listed.

Reply:
`_load_roman` creates the upper data strings, so their inclusion in the data class is redundant.

Reply:
Removed all upper-case Roman numerals from the data files.